File: order-service/src/main/java/com/petshop/orderservice/model/Address.java
package com.petshop.orderservice.model;

import lombok.Data;

@Data
public class Address {
    // Internal member variables
    private String firstName;
    private String lastName;
    private String address1;
    private String address2;
    private String city;
    private String state;
    private String zip;
    private String country;
    private String phone;
    private String email;
}

File: order-service/src/main/java/com/petshop/orderservice/model/Address.kt
package com.petshop.orderservice.model

data class Address(
    val firstName: String? = null,
    val lastName: String? = null,
    val address1: String? = null,
    val address2: String? = null,
    val city: String? = null,
    val state: String? = null,
    val zip: String? = null,
    val country: String? = null,
    val phone: String? = null,
    val email: String? = null
) {
}